tide/src/lib.rs

```diff
@@ -28,27 +25,19 @@ macro_rules! box_async {
 }
 
 #[macro_use]
-pub mod error;
+extern crate tide_core;
+
+#[cfg(feature = "cookies")]
+pub use tide_cookies as cookies;
```

Comment:
This should probably have `#[doc(inline)]` above it.

Reply:
Added.